packages/autocomplete/src/examples/multiselect.md

```diff
@@ -0,0 +1,304 @@
+            <Label {...getLabelProps()}>Advanced Multi-Select Example</Label>
+            <AutocompleteContainer
+              isOpen={state.isOpen}
+              tagFocusedKey={state.tagFocusedKey}
+              onSelect={selectedKey => {
+                let natoPhonetics = state.natoPhonetics;
+                const selectedKeys = Object.assign({}, state.selectedKeys);
+
+                if (selectedKeys[selectedKey]) {
+                  delete selectedKeys[selectedKey];
+                } else {
+                  selectedKeys[selectedKey] = true;
+                }
+
+                setState({ selectedKeys, natoPhonetics, inputValue: '' });
+
+                return true;
+              }}
+              onStateChange={newState => {
+                let inputValue = state.inputValue;
+
+                if (typeof newState.isOpen !== 'undefined' && !newState.isOpen) {
+                  newState.inputValue = '';
+                }
+
+                setState(newState);
+              }}
+              trigger={({
+                getTriggerProps,
+                getInputProps,
+                getTagProps,
+                triggerRef,
+                inputRef,
+                isOpen,
+                tagFocusedKey
+              }) => {
+                return (
+                  <FauxInput
+                    {...getTriggerProps({
+                      open: isOpen,
+                      small: true,
+                      select: true,
+                      tagLayout: true,
+                      focused: isOpen || typeof tagFocusedKey !== 'undefined' || state.isFocused,
+                      inputRef: ref => {
+                        this.wrapperRef = ref;
+                        triggerRef(ref);
+                      }
+                    })}
+                  >
+                    {renderTags(
+                      isOpen || typeof tagFocusedKey !== 'undefined' || state.isFocused,
+                      state.selectedKeys,
+                      tagFocusedKey,
+                      getTagProps
+                    )}
+
+                    <Input
+                      {...getInputProps(
+                        getFieldInputProps({
+                          bare: true,
+                          innerRef: inputRef,
+                          value: state.inputValue,
+                          onChange: e => {
+                            setState({ inputValue: e.target.value });
+                          },
+                          onKeyDown: e => {
+                            if (tagFocusedKey !== undefined) {
+                              if (
+                                e.keyCode === KEY_CODES.DELETE ||
+                                e.keyCode === KEY_CODES.BACKSPACE
+                              ) {
+                                deleteTag(tagFocusedKey);
+                              }
+
+                              // copy-to-clipboard functionality
```

Comment:
It would also be nice to show `cmd-A` functionality that selects all tags. See Gmail for the ability to delete all tags at once.

Reply:
> It would also be nice to show cmd-A functionality that selects all tags

This functionality isn't used anywhere in the products (that I am aware of). Also, not sure what the accessibility logic would be for this since you would then have to have the add/remove logic with shift+ctrl combinations as well.

Since this wasn't in the original I think this is out-of-scope for this iteration.

Reply:
Fair enough. Agreed out-of-scope.